go/app/server.go

```diff
@@ -180,6 +279,55 @@ func parseGetImageRequest(r *http.Request) (*GetImageRequest, error) {
 	return req, nil
 }
 
+type GetItemToIdResponse struct {
+	Name string `json:"name"`
+	Category string `json:"category"`
+	Image_Name string `json:"image_name"`
+}
+
+func (s *Handlers) GetItemToId(w http.ResponseWriter, r *http.Request) {
```

Comment:
GetItem**By**ID is better naming , i think!
```suggestion
func (s *Handlers) GetItemById(w http.ResponseWriter, r *http.Request) {
```